common/const.go

```diff
@@ -15,9 +15,8 @@
 package common
 
 const (
-	// RFC3339Milli is the RFC3339 with milliseconds for the default timestamp format
-	// log files.
-	RFC3339Milli = "2006-01-02T15:04:05.000Z07:00"
+	// RFC3339log is the RFC3339 timestamp format for log files.
+	RFC3339log = "2006-01-02T15:04:05Z07:00"
```

Comment:
`RFC3339Milli` was used because of the milliseconds granularly. If you want to use plain RFC3339, you have the `time.RFC3339`